src/Common/Commands.Common.Test/PowerBISettingsTests.cs

```diff
@@ -42,7 +41,7 @@ public void InValidEnvironmentInJson()
     }
   ],
   ""Settings"": {
-    ""Option1"": true
+    ""ForceDeviceCodeAuthentication"": true
```

Comment:
Maybe test ShowADALDebugMessages too.

Reply:
Wanted to test without or without the setting specified